src/openrct2/ride/rtd/gentle/FerrisWheel.h

```diff
@@ -59,7 +59,7 @@ constexpr RideTypeDescriptor FerrisWheelRTD =
         RatingsCalculationType::FlatRide,
         { RIDE_RATING(0, 60), RIDE_RATING(0, 25), RIDE_RATING(0, 30) },
         16,
-        0,
+        -1,
```

Comment:
If this `-1` has a special meaning, you should create a constant for it.

Reply:
I have extracted this into a prior commit, and then only applied the new constant to the new places on the second commit